cfgov/unprocessed/css/molecules/image-text-50-50.less

```diff
@@ -82,7 +82,6 @@
 */
 
 .m-image-text-50-50 {
-    padding-bottom: unit( @grid_gutter-width * 2 / @base-font-size-px, em );
```

Comment:
Can we do something like this: https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/unprocessed/css/organisms/sidebar-content.less?